gorevive/deps.bzl

```diff
@@ -0,0 +1,103 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_gazelle//:deps.bzl", "go_repository")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "new_git_repository")
+
+
+def go_revive_dependencies():
+    
+    _maybe(
+        go_repository,
+        name = "com_github_burntsushi_toml",
+        importpath = "github.com/BurntSushi/toml",
+        tag = "v0.3.0",
```

Comment:
Would be better to specify commit revisions here and below - they are immutable unlike tags.

Reply:
These deps are generated by Gazelle from https://github.com/mgechev/revive/blob/master/go.mod
I think it is safer to stick to whatever the author of revive has chosen (tags vs commits).
What do you think?

Reply:
Yeah, lets keep them in that case, that's fine.